docs/releases/3.0.txt

```diff
@@ -364,6 +364,17 @@ Requests and Responses
 * For use in, for example, Django templates, :attr:`.HttpRequest.headers` now
   allows look ups using underscores (e.g. ``user_agent``) in place of hyphens.
 
+.. _whats-new-security-3.0:
+
+Security
+~~~~~~~~
+
+* :setting:`SECURE_CONTENT_TYPE_NOSNIFF` setting now defaults to ``True``. With
+  the enabled :setting:`SECURE_CONTENT_TYPE_NOSNIFF`, the
+  :class:`~django.middleware.security.SecurityMiddleware` sets the
+  :ref:`x-content-type-options` header on all responses that do not already
+  have it.
```

Comment:
Should we also mention `X_FRAME_OPTIONS`? (see 05d0eca635853564c57e639ac5590674a7de2ed).